tests/testthat/test-derive_var_extreme_flag.R

```diff
@@ -27,7 +27,8 @@ input_worst_flag <- tibble::tribble(
   "TEST01", "PAT02", "PARAM03", "BASELINE", as.Date("2021-04-30"), 12.0
 )
 
-test_that("first observation for each group is flagged", {
+## Test 1: first observation for each group is flagged ----
```

Comment:
Thanks for updated these to follow our test guidance.  Is the big tibble above this used at all in these tests?  If not, can we delete it?

Reply:
I assumed it was in the previous tests but I suppose its actually not, going to go ahead and remove it